joomla/joomla-cms
# [5.2] Content Category Save to Menu

### Summary of Changes
This PR adds two new menu items to the save dropdown menu on the toollbar allowing you to save a categorey directly to a menu item - either as a blog or a list just as you can directly save an article to a menu item.

This is currently restricted to categories in the com_content component. It could be expanded to other components but I didn't see the use case


### Testing Instructions
Check that there are two new items in the save group as shown **only** for categories in com_content and not other places that categories are used such as banners and contacts

![image](https://github.com/user-attachments/assets/34109bb7-fc48-4530-a146-9d5100b98b1d)

Check that both new items work as expected and create blog or list menu items as shown for **both** new and existing categories

### Save to Menu as List
![image](https://github.com/user-attachments/assets/f0bf2b92-c518-40cc-8650-bbbcb961fdb9)

### Save to Menu as Blog
![image](https://github.com/user-attachments/assets/c0acbe42-7ddb-4949-a003-74607f050c00)


### Link to documentations
Please select:
